vendor/bat-native-ads/src/bat/ads/internal/eligible_ads/round_robin_advertisers.h

```diff
@@ -22,8 +23,10 @@ T FilterSeenAdvertisers(const T& ads,
   const auto iter =
       std::remove_if(unseen_advertisers.begin(), unseen_advertisers.end(),
                      [&seen_advertisers](CreativeAdInfo& ad) {
-                       return seen_advertisers.find(ad.advertiser_id) !=
-                              seen_advertisers.end();
+                       SplitTestFrequencyCap frequency_cap;
+                       return frequency_cap.ShouldExclude(ad) ||
```

Comment:
If an ad is excluded due to a split test it should not be considered for the round robin